src/App.js

```diff
@@ -1,25 +1,68 @@
 import React from 'react';
+import { Clock } from './Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+class App extends React.Component {
+  state = {
+    time: new Date().toLocaleTimeString(),
+    isClockVisible: true,
+    clockName: 0,
+  }
+
+  componentDidMount() {
+    setInterval(() => {
+      if (!this.state.isClockVisible) {
+        return;
+      }
+
+      this.setState({ time: new Date().toLocaleTimeString() });
+      // eslint-disable-next-line
+      console.log(this.state.time);
+    }, 1000);
+  }
+
+  showClock = () => {
+    this.setState({ isClockVisible: true });
+  }
+
+  hideClock = () => {
+    this.setState({ isClockVisible: false });
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <h1>React clock</h1>
+        <Clock
+          time={this.state.time}
+          visibility={this.state.isClockVisible}
+          name={this.state.clockName}
```

Comment:
можно было бы перед return сделать деструктуризацию
```suggestion
         { time, isClockVisible, ClockName } = this.state
```
но это как я понимаю по желанию и кому как удобнее